master/reference/public-cloud/digital-ocean.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Calico on Digital Ocean
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as Digital Ocean.
+
+## How to Run Calico in Digital Ocean
+
+Digital Ocean offers two networking options for droplets.
+Calico can be launched on either:
+
+- Public Interface
+- Private Interface [Optional]
```

Comment:
In digital ocean, all droplets have a public interface, but you can optionally add a private interface. I'll clarify that that's what I'm referring to here.